### PR TITLE
Add setting to configure the rate at which to refresh clients 

### PR DESCRIPTION
**Pull request by romac**
Closes: #3397

## Description

Add a `client_refresh_rate` setting to specify the rate at which to refresh clients referencing this chain, as a fraction of their trusting period.

______

### PR author checklist:
- [x] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules).
- [x] Linked to GitHub issue.
- [x] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: crates/relayer/src/worker/client.rs [hunk 1]
```diff
@@ -35,34 +34,23 @@ pub fn spawn_refresh_client<ChainA: ChainHandle, ChainB: ChainHandle>(
...
+        Some(REFRESH_CHECK_INTERVAL),
         move || {
-            // This is used for integration tests until `spawn_background_task`
-            // uses async instead of threads
-            if Instant::now() < next_refresh {
-                return Ok(Next::Continue);
-            }
-
-            // Use retry mechanism only if `client.refresh()` fails.
+            // Try to refresh the client, but only if the refresh window as expired.
```
Comment (ljoss17) [inline crates/relayer/src/worker/client.rs:46]:
```suggestion
            // Try to refresh the client, but only if the refresh window has expired.
```

**Review by ljoss17 — CHANGES_REQUESTED** [review]
Looks good, just some small nits

File: crates/relayer/src/config.rs [hunk 2]
```diff
@@ -196,6 +198,15 @@ pub mod default {
         Byte::from_bytes(33554432)
     }
 
+    pub fn trust_threshold() -> TrustThreshold {
+        TrustThreshold::ONE_THIRD
```
[the remark this answers is not in the document]
Comment (romac) [reply crates/relayer/src/config.rs:202]: Good catch!

**Review by ljoss17 — APPROVED** [review]
Looks good to me!!

**Pull request merged**